website/server/controllers/api-v3/tasks.js

```diff
@@ -901,11 +901,19 @@ api.clearCompletedTodos = {
       userId: user._id,
```

Comment:
Could you please also modify the comment a couple of lines above here so that it says "Do not delete challenge and group completed ..."? Sorry, I realise that's not your omission!